### DISCUSSION TITLE
Git LFS is disabled for this repository

### DISCUSSION THREAD

**Discussion by jonunji** [community/community#32455] [New to GitHub]
When I try to push my changes on the GitHub desktop client, I get the following message:

```
batch response: Git LFS is disabled for this repository.
Uploading LFS objects:   0% (0/57), 0 B | 0 B/s, done.
error: failed to push some refs to 'https://github.com/jonunji/Mousebound.git'
```

I believe there is an issue with my account or something.
Thanks!

**Comment by queenofcorgis** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Good luck :)

**Comment by HossamMuhammedOmar** [comment]
You just need to contact GitHub support [support.github.com/contact](https://support.github.com/contact)
They will solve your issue.
